src/ToolBox/SOS/lldbplugin/services.cpp

```diff
@@ -8,6 +8,8 @@
 #include <string.h>
 #include <string>
 
+#define CONVERT_FROM_SIGN_EXTENDED(offset) ((ULONG)(offset))
```

Comment:
Won't this break 64 bit platforms? Maybe I don't reminder my C++ casting behavior but won't this strip off the high dword of a 64 bit address?

Reply:
Thank you for suggestion! ULONG is unsigned int and such cast strips off the high dword of a 64 bit address. I changed it to ULONG_PTR that is unsigned long.